backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -176,11 +182,15 @@ public Boolean setupNativeIntegration(BacktraceBase client, BacktraceCredentials
         if (minidumpSubmissionUrl == null) {
             return false;
         }
-        // Path to Crashpad native handler
-        String handlerPath = _applicationContext.getApplicationInfo().nativeLibraryDir + _crashpadHandlerName;
-        if (!FileHelper.isFileExists(handlerPath)) {
-            return false;
-        }
+
+        // Create the crashpad directory if it doesn't exist
+        String databasePath = getSettings().getDatabasePath() + _crashpadDatabasePathPrefix;
```

Comment:
extract this block of code to method `createCrashpadDirectory`

btw what if this directory already exists?